### PR TITLE
Do not generate and pack runtime config for libraries

### PR DESCRIPTION
**Pull request by pakrym**
Fixes: https://github.com/dotnet/cli/issues/2396

/cc @davidfowl

---
This change is [Reviewable](https://reviewable.io/reviews/dotnet/cli/2401)

### CODE REVIEW

File: src/Microsoft.DotNet.Compiler.Common/Executable.cs [hunk 1]
```diff
@@ -109,8 +112,11 @@ private void CopyAssets(IEnumerable<LibraryExport> libraryExports)
         private void WriteDepsFileAndCopyProjectDependencies(LibraryExporter exporter)
         {
             WriteDeps(exporter);
-            WriteRuntimeConfig(exporter);
-            WriteDevRuntimeConfig(exporter);
+            if (!_context.ProjectFile.HasRuntimeOutput(_configuration))
+            {
+                WriteRuntimeConfig(exporter);
+                WriteDevRuntimeConfig(exporter);
```
Comment (livarcocc) [inline src/Microsoft.DotNet.Compiler.Common/Executable.cs]: @schellap Please, review this. This breaks our initial assumption that you would always have a .dev.config file. Is this an ok change to make?

Comment (livarcocc) [inline src/Microsoft.DotNet.Compiler.Common/Executable.cs]: @schellap Please, review this. This breaks our initial assumption that you would always have a .dev.config file. Is this an ok change to make?

**Comment by davidfowl** [comment]
:shipit:

**Comment by pakrym** [comment]
@dotnet-bot test RHEL7.2 x64 Release Build please

**Pull request merged**